<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="158" started-at="2018-06-14T09:32:09Z" finished-at="2018-06-14T09:32:09Z">
    <groups>
    </groups>
    <test name="First class" duration-ms="39" started-at="2018-06-14T09:32:09Z" finished-at="2018-06-14T09:32:09Z">
      <class name="dd_tests.ValidateIMEI">
        <test-method status="PASS" signature="isskip()[pri:0, instance:dd_tests.ValidateIMEI@3c5a99da]" name="isskip" is-config="true" duration-ms="27" started-at="2018-06-14T15:02:09Z" finished-at="2018-06-14T15:02:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isskip -->
        <test-method status="FAIL" signature="enterIMEI(java.lang.String)[pri:0, instance:dd_tests.ValidateIMEI@3c5a99da]" name="enterIMEI" duration-ms="0" started-at="2018-06-14T15:02:09Z" data-provider="getData" finished-at="2018-06-14T15:02:09Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] dd_tests.ValidateIMEI.getData() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] dd_tests.ValidateIMEI.getData() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:155)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterIMEI -->
      </class> <!-- dd_tests.ValidateIMEI -->
    </test> <!-- First class -->
    <test name="Second class" duration-ms="5" started-at="2018-06-14T09:32:09Z" finished-at="2018-06-14T09:32:09Z">
      <class name="dd_tests.InspectionPage">
        <test-method status="PASS" signature="init()[pri:0, instance:dd_tests.InspectionPage@58ceff1]" name="init" is-config="true" duration-ms="58096" started-at="2018-06-14T15:01:11Z" finished-at="2018-06-14T15:02:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="SKIP" signature="isskip()[pri:0, instance:dd_tests.InspectionPage@58ceff1]" name="isskip" is-config="true" duration-ms="0" started-at="2018-06-14T15:02:09Z" finished-at="2018-06-14T15:02:09Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Run Mode Set as No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Run Mode Set as No
	at dd_tests.InspectionPage.isskip(InspectionPage.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isskip -->
        <test-method status="FAIL" signature="Inspection(java.lang.String)[pri:0, instance:dd_tests.InspectionPage@58ceff1]" name="Inspection" duration-ms="0" started-at="2018-06-14T15:02:09Z" data-provider="getData" finished-at="2018-06-14T15:02:09Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[] dd_tests.InspectionPage.getData() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[] dd_tests.InspectionPage.getData() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:155)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Inspection -->
      </class> <!-- dd_tests.InspectionPage -->
    </test> <!-- Second class -->
  </suite> <!-- Suite -->
</testng-results>
